spec/controllers/languages_controller_spec.rb

```diff
@@ -10,6 +10,13 @@
         get :index
         expect(response).to render_template("index")
       end
+
+      it "displays language names and codes" do
+        create(:language, name: "Deutsch", short: "de")
+        get :index
+        expect(response.body).to include("Deutsch")
+        expect(response.body).to include("(de)")
+      end
```

Comment:
We generally prefer to check the content of view files in feature tests. A better place for this check might be at the end of this feature test: https://github.com/otwcode/otwarchive/blob/b6c964443004cac7ebf04664ba08b1618d1a107d/features/admins/admin_languages.feature#L6-L17